review_checker/views/android_clients.view.lkml

```diff
@@ -0,0 +1,107 @@
+include: "//looker-hub/review_checker/views/android_clients.view.lkml"
+
+view: +android_clients {
+
+  dimension: client_id {
+    #primary_key: yes
+    sql: ${TABLE}.client_id ;;
+    hidden: yes
+  }
+
+  dimension: client_id_date {
+    primary_key: yes
+    sql: CONCAT(${TABLE}.client_id, ${TABLE}.submission_date) ;;
+    type: string
+  }
+
+  #measures
+  measure: active_hours {
```

Comment:
many of these measures are dimensions on the parent view, should you hide those?

Reply:
a number are used as filters, but I've hidden those that I think are less likely to be used as a filter.